### ISSUE TITLE
Gutenberg: v10.9.x plugin upgrade

### ISSUE THREAD

**Issue by fullofcaffeine** [Automattic/wp-calypso#53725]
Previous upgrade: https://github.com/Automattic/wp-calypso/issues/53394.
Release notes: [v10.9.0-rc.1](https://github.com/WordPress/gutenberg/releases/tag/v10.9.0-rc.1), [v10.9.0](https://github.com/WordPress/gutenberg/releases/tag/v10.9.0), [v10.9.1](https://github.com/WordPress/gutenberg/releases/tag/v10.9.1).

- [x] Install and activate
  - [x] Copy & install v10.9.0-rc.1 (D62999-code, D63086-code)  (It's a custom build with two additional fixes, see the description for the install diff)
  - [x] Activate v10.9.0-rc.1 on edge (D63088-code)
  - [x] Copy & install v10.9.0 (D63254-code, D63255-code)
  - [x] Activate v10.9.0 on edge (D63256-code)
  - [x] Activate v10.9.0 on production (D63089-code)
  - [x] Copy and install v10.9.1 (D63347-code, D63352-code)
  - [x] Activate v10.9.1 on edge (D63353-code)
  - [x] Activate v10.9. on production (D63354-code)
  - [x] Request AT upgrade
- [x] Testing
  - [x] Desktop viewport E2E 
  - [x] Mobile viewport E2E
  - [x] WPCOM (D63089-code, D63354-code)
- [x] Publish internal announcements
- [ ] Clean up unused releases
- [x] Add log entry to the Gutenberg Upgrade Log [pcoGjb-g-p2]
- [x] Open a new issue for the next upgrade, transfer remaining tasks, close this issue.

### Other issues 🐛

- [x] `jetpack/subscription` block is triggering a lot of invalidation messages.
    - Discussed in: p1623796237416100/1623793937.415200-slack-C7YPUHBB2 and p1623795437120700-slack-CBG1CP4EN
    - This seems benign so far. @glendaviesnz will have a look into it.
    - Fixed in: https://github.com/WordPress/gutenberg/pull/32742

#### Transfered from previous release(s)

- [x] `wp-components`'s `index.js` is broken by the WPCOM minifier. There's a hotfix (D61090-code, D61091-code) applied in the RC1 that allowed us to ship it to simple sites for now. We still don't know what exactly in v10.6.0 RC1 caused this. We might need to be hotfixing on simple sites while we don't have a perm. solution. See: p1620321684463200-slack-C02A41GCS and p1620319065221200-slack-C4GAP2RHD.
  - [x] https://github.com/WordPress/gutenberg/pull/31732
- [x] P2 inline comments broken (and potentially other P2-related issues). Reported here: p1620327615008700-slack-C025Q5RK2  cc @johngodley (P2s have been pinned to 10.5.4 for now, see: p1621258202222400/1620838719.129600-slack-C7YPUHBB2, which makes this issue not a blocker for now).

---
cc @inaikem @simison

**Comment by fullofcaffeine** [comment]
Starting from 10.9, Gutenberg production builds include unminified and minified JS assets (introduced in https://github.com/WordPress/gutenberg/pull/31732). For some reason, unminified jS assets have `CRLF` and `LF` bytes in them. This causes `svn` to complain in WPCOM (see: pMz3w-di5-p2#comment-89175). 

We still don't know if this issue was already present and masked by the Terser plugin in Gutenberg, or if it was introduced in https://github.com/WordPress/gutenberg/pull/31732. We'll investigate soon. For now, a workaround has been documented here: pMz3w-di5-p2#comment-89229, and should be done before we create the Gutenberg install diff in WPCOM.

**Comment by fullofcaffeine** [comment]
This is done. Next one: https://github.com/Automattic/wp-calypso/issues/54180.
